cluster/ubuntu/build.sh

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Download the etcd, flannel, and K8s binaries automatically
+# Download the etcd, flannel, and K8s binaries automatically and stored in binaries directory
```

Comment:
You don't need to build flannel. Just grab the flannel release binary as with all other tools:
https://github.com/coreos/flannel/releases/download/v0.4.0/flannel-0.4.0-linux-amd64.tar.gz